generator/README.md

```diff
@@ -0,0 +1,14 @@
+## :generator
+
+This module holds a little Gradle plugin whose sole purpose is to provide
+[:core](../core) with the `generateKeywordsSrc` task.
+
+This task, generates three enum classes: `HardKeywords`, `ModifierKeywords`, and `SoftKeywords`.
+These enums together contain all restricted Kotlin keywords to be taken into account when generating our own
+code in Notebooks or any of our [plugins](../plugins). Words like "package", "fun", "suspend", etc...
+
+As the Kotlin language can change over time, this task ensures that any changes to the language
+will be reflected in our code generation.
+
+This module can probably be moved under [:plugins](../plugins):
```

Comment:
Do you want to keep this line?

Reply:
Hmm I don't think it would hurt. Informs people that it is a plugin and likely belongs there but we just haven't come around to moving it yet.